planetiler-core/src/main/java/com/onthegomap/planetiler/config/PlanetilerConfig.java

```diff
@@ -166,6 +167,7 @@ public static PlanetilerConfig from(Arguments arguments) {
         "Planetiler downloader (https://github.com/onthegomap/planetiler)"),
       arguments.getDuration("http_timeout", "Timeout to use when downloading files over HTTP", "30s"),
       arguments.getInteger("http_retries", "Retries to use when downloading files over HTTP", 1),
+      arguments.getLong("http_retry_wait", "How long (in milliseconds) to wait before retrying HTTP request", 5_000),
```

Comment:
Thanks for adding this as a config option! Let's use the `getDuration` helper though which handles unit conversions.

Reply:
Ah, overlooked that, thanks. Adjusted.